src/client/search/AddSearchTerm.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { addSearchTerm } from './SearchActions';
+
+let AddSearchTerm = ({ dispatch, addSerchTermRef }) => {
+  const keywordSearchStyle = {
+    display:'none',
+    paddingLeft:20,
+    paddingTop:10,
+    marginTop:10,
+    borderTop:1,
+    borderTop: '2px dashed #D3D5D8',
+  };
+  return (
+    <div className="ui fluid big transparent input"
+      style={keywordSearchStyle}
+      id="searchTermContainer">
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2226118788/issues/source?bid=3148782&fileBranchId=3174638#l17)